source/extensions/filters/common/expr/context.h

```diff
@@ -77,6 +79,63 @@ constexpr absl::string_view Upstream = "upstream";
 constexpr absl::string_view UpstreamLocalAddress = "local_address";
 constexpr absl::string_view UpstreamTransportFailureReason = "transport_failure_reason";
 
+// Enumeration of all properties. Any new property symbol must be added here.
```

Comment:
Does `here` means to one of the lists in the `WrapperFieldValues` class?

Reply:
Correct, it's referring to the class below. Can I make it more clear or this is fine?